terraform/modules/elasticsearch_broker/security_group.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_security_group" "elasticsearch" {
+  description = "Allow access to incoming elasticsearch traffic"
+  vpc_id = "${var.vpc_id}"
+
+  ingress {
+    from_port = 443
+    to_port = 443
+    protocol = "tcp"
+    security_groups = ["${var.security_groups}"]
+  }
+
+  egress {
+    from_port = 0
+    to_port = 0
+    protocol = "-1"
+    security_groups = ["${var.security_groups}"]
```

Comment:
if `${var.security_groups}` is a `list`, do we need the braces here, or will that resolve to `[[group, other-group],]`?